manim_slides/present.py

```diff
@@ -956,9 +997,15 @@ def present(
     if skip_all:
         exit_after_last_slide = True
 
+    presentation_configs = get_scenes_presentation_config(scenes, folder)
+
+    if resolution:
+        for presentation_config in presentation_configs:
+            presentation_config.resolution = resolution
+
```

Comment:
@Fairlight8 Fifth, update the configs `if resolution is not None`*

*Previously it was `if resolution:`, but testing `if not None is better` (because some values are also fasly, e.g., 0).